<?xml version="1.0" encoding="UTF-8"?>
<ui version="5.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vecty</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../beam-studio/public/icon.png</normaloff>../beam-studio/public/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QQuickWidget" name="quickWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="resizeMode">
       <enum>QQuickWidget::SizeRootObjectToView</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Layer</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListWidget" name="layerList">
          <property name="movement">
           <enum>QListView::Free</enum>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addLayer">
            <property name="text">
             <string>Add Layer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Parameters&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="parameterArea">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>258</width>
             <height>210</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QComboBox" name="comboBox_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string>Wood 3mm</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Acrylic 3mm</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Power</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_2">
                <property name="suffix">
                 <string> %</string>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>25</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>25</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Speed</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_3">
                <property name="suffix">
                 <string> mm/s</string>
                </property>
                <property name="maximum">
                 <number>300</number>
                </property>
                <property name="value">
                 <number>300</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_2">
              <property name="maximum">
               <number>300</number>
              </property>
              <property name="value">
               <number>300</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Execute</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox">
                <property name="suffix">
                 <string> times</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Object</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionClear"/>
    <addaction name="actionGroup"/>
    <addaction name="actionUngroup"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #F0F0F0;
border-width: 0px;
spacing: 8px;</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea|Qt::RightToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>38</width>
     <height>30</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect"/>
   <addaction name="actionPhoto"/>
   <addaction name="actionText"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionOval"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+X</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../qml.qrc">
     <normaloff>:/images/icon-cursor-01.png</normaloff>
     <normalon>:/images/select.png</normalon>:/images/icon-cursor-01.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>Select (&amp;V)</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="icon">
    <iconset resource="../../qml.qrc">
     <normaloff>:/images/icon-rect-01.png</normaloff>:/images/icon-rect-01.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="toolTip">
    <string>Draw rectangles</string>
   </property>
  </action>
  <action name="actionOval">
   <property name="icon">
    <iconset resource="../../qml.qrc">
     <normaloff>:/images/icon-oval-01.png</normaloff>:/images/icon-oval-01.png</iconset>
   </property>
   <property name="text">
    <string>Oval</string>
   </property>
   <property name="toolTip">
    <string>Draw ovals</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionPhoto">
   <property name="icon">
    <iconset resource="../../qml.qrc">
     <normaloff>:/images/icon-photo-01.png</normaloff>:/images/icon-photo-01.png</iconset>
   </property>
   <property name="text">
    <string>Photo</string>
   </property>
   <property name="toolTip">
    <string>Image</string>
   </property>
  </action>
  <action name="actionText">
   <property name="icon">
    <iconset resource="../../qml.qrc">
     <normaloff>:/images/icon-text-01.png</normaloff>:/images/icon-text-01.png</iconset>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
   <property name="toolTip">
    <string>Text</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionGroup">
   <property name="text">
    <string>Group</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionUngroup">
   <property name="text">
    <string>Ungroup</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+G</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../qml.qrc"/>
 </resources>
 <connections/>
</ui>
